### PR TITLE
[PKG-1994] upgrade to 2.11 for icu-73.1

### PR DESCRIPTION
**Pull request by rafaelmartins**
Depends on:
- https://github.com/AnacondaRecipes/icu-feedstock/pull/3

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/P/PyICU/PyICU-{{ version }}.tar.gz
-  sha256: c0655302e2aea16f9acefe04152f74e5d7d70542e9e15c89ee8d763c8e097f56
+  sha256: 3ab531264cfe9132b3d2ac5d708da9a4649d25f6e6813730ac88cf040a08a844
 
 build:
-  number: 0
+  number: 1
```
Comment (boldorider4) [inline recipe/meta.yaml:12]: if this is a package update (from older to newer version) build number should be reset to 0.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,7 +32,10 @@ test:
   imports:
     - icu
   commands:
+    - pip check
     - python -c "from icu import Locale;locale = Locale('pt_BR')"
```
Comment (boldorider4) [inline recipe/meta.yaml:36]: is there a way to add this to `test.py`? I think you can add a `files:` section to the test section and commit these tests in a separate file.

File: recipe/meta.yaml (hunk 1 above)
Comment (rafaelmartins) [reply recipe/meta.yaml]: true, this started as a rebuild, but ended up as an upgrade and I forgot to revisit build number

File: recipe/meta.yaml (hunk 2 above)
Comment (rafaelmartins) [reply recipe/meta.yaml:36]: I don't think it is worth converting this single line test to a separate file.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ requirements:
   host:
     - python
     - pip
-    - icu
+    - icu {{ icu }}
```
[the remark this answers is not in the document]
Comment (rafaelmartins) [reply recipe/meta.yaml:24]: this is mentioned in the original icu pr, but I'll revisit the prs next week marking them as draft

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**